files/zh-cn/web/api/document/createtextnode/index.html

```diff
@@ -32,10 +32,7 @@ <h2 id="示例">示例</h2>
 &lt;/head&gt;
 
 &lt;body&gt;
-  &lt;button value="好耶！"&gt;好耶！&lt;/button&gt;
-  &lt;button value="坏耶！"&gt;坏耶！&lt;/button&gt;
-  &lt;button value="Rikka! "&gt;Rikka!&lt;/button&gt;
-  &lt;button value="日卡卡！"&gt;日卡卡！&lt;/button&gt;
+  &lt;button value="createTextNode"&gt;createTextNode&lt;/button&gt;
```

Comment:
```
  <button onclick="addTextNode('YES! ');">YES!</button>
  <button onclick="addTextNode('NO! ');">NO!</button>
  <button onclick="addTextNode('WE CAN! ');">WE CAN!</button>
```
Please update according the latest content.

Please describe what you done in the title, not just Update index.html.